pyvista/core/filters/data_set.py

```diff
@@ -824,7 +824,7 @@ def threshold_percent(dataset, percent=0.50, scalars=None, invert=False,
         >>> threshed = grid.threshold_percent(0.8)
         >>> threshed.plot(cmap='gist_earth_r', show_scalar_bar=False, show_edges=True)
 
-        See :ref:`common_filter_example` for more examples using a similar filter.
+        See :ref:`common_filter_example` for more examples using this filter.
```

Comment:
I think we changed this in the dataset filters PR because the linked example contains `threshold` rather than `threshold_percent`.

Reply:
Reverted in 55ba3daa9ec943acb388ac83608d7bd7ca5341b1.